### PR TITLE
add cask file for MathType app 

### PR DESCRIPTION
**Pull request by daehyunpy**
add cask file: mathtype.rb

I’m wondering why the app doesn’t work well if I installed the app using `pkg`. 
This cask file try to install it using `manual`. 
Is anyone here has this situation before?

### CODE REVIEW

**Comment by vitorgalvao** [comment]
`installer :manual` is exclusively for cases with app installers. If `pkg` doesn’t work here, we fix that. What’s the error?

**Comment by hyuna917** [comment]
@vitorgalvao oh, i’m sorry. i just ran `cleanup` and reinstalled. than, it works. thank you.

File: Casks/mathtype.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'mathtype' do
+  version '6.7'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/mathtype.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/mathtype.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  url "http://www.dessci.com/en/dl/MTM#{version.gsub('.', '')}h_EN.pkg"
+  name 'MathType'
+  homepage 'www.dessci.com'
+  license :commercial
+
+  pkg "MTM#{version.gsub('.', '')}h_EN.pkg"
+  uninstall :pkgutil => "com.dessci.mathtype#{version.gsub('.', '')}Hf.MathType.pkg",
```
Comment (vitorgalvao) [inline Casks/mathtype.rb]: Needs a blank line above. Also, structure needs to be fixed:

```
uninstall :pkgutil => "com.dessci.mathtype#{version.gsub('.', '')}Hf.MathType.pkg",
              :delete => "/Applications/MathType #{version.to_i}/MathType #{version.to_i} Install Log.txt",
              :rmdir => "/Applications/MathType #{version.to_i}"
```

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ab3241e5f23c161863c6b0fc8d4c72b8cefc883e. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by DannyNemer** [comment]
This cask installation fails and I believes requires `installer :manual`, which does not fail, instead of `pkg`.

The following occurs when attempting to open `MathType.app` following this cask's installation:


The missing "configuration file" is `/Applications/MathType 6/.MT6.dsc`. This file is normally created by the following manual installation step:

**Pull request closed without merging**